.codeclimate.yml

```diff
@@ -0,0 +1,19 @@
+ratings:
+  paths: 
+  - "**/*.go"
+
+engines:
+  fixme:
+    enabled: true
+    config:
+      strings:
```

Comment:
What about TODO?

Reply:
I included and it gave me a bunch of errors for every TODO in the code, which are not things that can be fixed so easily. So I took it out, at least until we can go over the list. We should do the TODOs, but also shouldn't feel discouraged to mark things with TODO comments in the code for things which are not urgent just because codeclimate is going to complain.